cmd/oc-mirror/main.go

```diff
@@ -1,11 +1,21 @@
 package main
 
 import (
+	"github.com/containers/buildah"
+	"github.com/containers/storage/pkg/unshare"
 	"github.com/openshift/oc-mirror/pkg/cli/mirror"
 	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
 func main() {
+	// oc-mirror runs in rootless mode. It is therefore necessary to
+	// ensure that oc-mirror is re-executed in a user namespace where
+	// it has root privileges.
+	if buildah.InitReexec() {
```

Comment:
@sherine-k - should we not gate this ? i.e move it to the executor.go , and only set this if we have graph: true set